server/cmwell-ws/app/security/AuthCache.scala

```diff
@@ -13,59 +13,46 @@
   * limitations under the License.
   */
 
-
 package security
 
-import java.util.concurrent.TimeUnit
-
 import cmwell.domain.{Everything, FileInfoton}
-import com.google.common.cache.{CacheLoader, CacheBuilder, LoadingCache}
+import cmwell.zcache.L1Cache
+import com.typesafe.scalalogging.LazyLogging
 import logic.CRUDServiceFS
-import play.api.libs.json.{Json, JsValue}
+import play.api.libs.json.{JsValue, Json}
 
 import scala.concurrent.Await
-import scala.concurrent.duration.Duration
-import scala.util.{Success, Try}
 import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration._
+import scala.util.Try
 
 /**
- * Created by yaakov on 1/20/15.
- */
-object AuthCache {
+  * Created by yaakov on 1/20/15.
+  */
+object AuthCache extends LazyLogging {
   private val usersFolder = "/meta/auth/users"
   private val rolesFolder = "/meta/auth/roles"
 
-  def getUserInfoton(username: String): Option[JsValue] =
-    Try(userInfotonsCache.get(username)).toOption
+  // TODO Do not Await.result... These should return Future[Option[JsValue]]]
 
-  def invalidateUserInfoton(username: String) = userInfotonsCache.invalidate(username)
+  def getUserInfoton(username: String): Option[JsValue] =
+    Try(Await.result(usersCache(username), 3.seconds)).toOption.flatten
```

Comment:
Normally I'd say yes, nothing should be hard-coded... But not this time:

1. This value is how long we wait for CAS to respond, having it different per environment may do some sense, but it's not crucial, becuase we hope any direct read by path should be way less than `3.seconds`, for any environment.
2. This `Await.result` will be very soon gone... So having it in config will just cause noise.